### PR TITLE
refactor: Pass the permanent download flag via GraphQL variables

### PR DESCRIPTION
**Pull request by olivereisenhut**
Title says it all.

### CODE REVIEW

File: src/Api.ts [hunk 1]
```diff
@@ -186,22 +186,20 @@ export async function requestAssetsById(
     { domain, bearerToken, permanentDownloadUrls }: Options,
     ids: Asset[],
 ): Promise<FrontifyAsset[]> {
-    const response = await httpCall(`https://${domain}/graphql`, {
+    const response = (await httpCall(`https://${domain}/graphql`, {
         method: 'POST',
         headers: {
             'content-type': 'application/json',
             authorization: `Bearer ${bearerToken}`,
-            'x-frontify-beta': 'enabled',
```
Comment (Magarif) [inline src/Api.ts:194]: Why removing beta here?

**Review by Magarif — COMMENTED** [review]
Looks good 👍 Just not sure why we remove beta header?

File: src/Api.ts (hunk 1 above)
Comment (olivereisenhut) [reply src/Api.ts:194]: It's not needed at the moment. I added it for upcoming features.

**Review by jprats89 — APPROVED** [review]
Not clue what's about, but since Petar was ok with it here you have my approval. Let me know if do you want to me to challenge it.

**Pull request merged**